examples/jsm/controls/OrbitControls.js

```diff
@@ -189,6 +189,10 @@ class OrbitControls extends EventDispatcher {
 
 				offset.copy( position ).sub( scope.target );
 
+				// so camera.up is the orbit axis
+				quat.setFromUnitVectors( object.up, yAxisUp );
+				quatInverse = quat.clone().invert();
```

Comment:
The `clone()` usage still creates unnecessary instances.

When fixed, the code is actually like #18829. I don't think we need two open PRs for the same thing with essentially the same implementation.

Reply:
Ok, fixed. Shoud we close this PR, and wait #18828 to merge?

Reply:
Yes, I think that is better. I've just rebased #18829 so it can be merged if approved.